homework_30_11/src/Triangle/Triangle.java

```diff
@@ -0,0 +1,89 @@
+package Triangle;
+import java.util.Scanner;
+/*Написать класс Triangle имитирующий прямоугольный треугольник, со сторонами a,b,c и методами получения величины каждой
+    стороны, площади, периметра треугольника. а так же добавить метод проверяющий выполнение теоремы пифагора.
+    Если стороны заданы не верно вывести сообщение на екран.*/
+public class Triangle {
+    double sideA;
+    double sideB;
+    double sideC;
+
+    public Triangle(double sideA,double sideB,double sideC) {
+        this.sideA=sideA;
+        this.sideB=sideB;
+        this.sideC=sideC;
+    }
+
+   /* public double getSideA() {
+        Scanner inputScanner = new Scanner(System.in);
+        double sideA = inputScanner.nextDouble();
+        System.out.print("Please, enter side A of the triangle, sm: ");
+        return getSideA();
+    }
+
+    public double getSideB() {
+        Scanner inputScanner = new Scanner(System.in);
+        double sideB = inputScanner.nextDouble();
+        System.out.print("Please, enter side B of the triangle, sm: ");
+        return getSideB();
+    }
+
+    public double getSideC() {
+        Scanner inputScanner = new Scanner(System.in);
+        System.out.print("Please, enter side C of the triangle, sm: ");
+        double sideC = inputScanner.nextDouble();
+        return getSideC();
+    }*/
+
+    public void getTrianglePerimeter() {
+        System.out.println("Perimeter of the triangle is: " + (sideA + sideB + sideC) + " sm.");
+    }
+
+    public void getTriangleSquare() {
+        System.out.println("Square of of the triangle is: " + ((sideA * sideB) / 2) + " sq.sm.");
+    }
+
+    public void isPythagorTheoremOk() {
+        if (sideA > sideB && sideA > sideC) {
+            if ((Math.pow(sideA, 2) == ((Math.pow(sideB, 2) + Math.pow(sideC, 2))))) {
+                System.out.println("The Pythagor theorem is ok.");
+            }else{
+                System.out.println("It isn`t a right triangle.");
+            }
+        }else if(sideB > sideA && sideB > sideC) {
+            if ((Math.pow(sideB, 2) == ((Math.pow(sideA, 2) + Math.pow(sideC, 2))))) {
+                System.out.println("The Pythagor theorem is ok.");
+            } else {
+                System.out.println("It isn`t a right triangle.");
+            }
+        }else if(sideC > sideA && sideC > sideB) {
+            if ((Math.pow(sideC, 2) == ((Math.pow(sideA, 2) + Math.pow(sideB, 2))))) {
```

Comment:
I am not agree because the task is: user enters 3 sides. We can`t concider which is it hypotenuse. Because it doesn`t matter. If triangle has hypotenuse, of course, it is right triangle. 

What is my fault? 
Also we check the theorem of Pythagor for decision that the triangle  is rigth. :)

Reply:
if you want to use overcomplicacted solution leave it as it is.